resources/measures/add_blinds_to_selected_windows/tests/add_blinds_to_selected_windows_test.rb

```diff
@@ -78,108 +124,134 @@ def run_test(test_name, osm_path, epw_path, blinds_expected)
     assert(File.exist?(run_dir(test_name)))
 
     # change into run directory for tests
+    start_dir = Dir.pwd
```

Comment:
remove the `start_dir = Dir.pwd`; we don't want to change directories in the test. that causes issues.